src/index.css

```
@@ -0,0 +1,99 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+@font-face {
+    font-family: 'spotify-circular-cyrillic';
+    src: url(https://open.scdn.co/cdn/fonts/CircularSpUICyrOnly-Book.6a858994.woff2);
+}
+
+.app {
+    display: grid;
+    height: 100vh;
+    overflow: hidden;
+    grid-template-columns: 241px 1fr;
+    grid-template-rows: min-content 1fr min-content;
+    font-family: 'spotify-circular-cyrillic', Helvetica, Arial, sans-serif;
+}
+
+.header {
+    display: flex;
+    align-items: center;
+    padding: 25px 25px;
+    background-color: #000000;
```

Рекомендация
Можно сократить до:
```suggestion
    background-color: #000;
```